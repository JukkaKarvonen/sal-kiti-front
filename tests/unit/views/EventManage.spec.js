import { shallowMount, createLocalVue } from "@vue/test-utils";
import EventManage from "@/views/EventManage.vue";

import BootstrapVue from "bootstrap-vue";

const localVue = createLocalVue();
localVue.use(BootstrapVue);

describe("EventManage.vue", () => {
  it("renders content", () => {
    const wrapper = shallowMount(EventManage, {
      localVue
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
