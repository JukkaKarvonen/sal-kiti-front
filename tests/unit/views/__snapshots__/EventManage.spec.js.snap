// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventManage.vue renders content 1`] = `
<div
  class="event"
>
  <eventform-stub />
</div>
`;
